_overviews/scala3-book/taste-control-structures.md

```diff
@@ -231,29 +235,26 @@ It’s one-line syntax looks like this:
 while x >= 0 do x = f(x)
 ```
 
-Again, Scala’s control structure syntax is flexible, and you can write this code in different ways depending on your preferences:
+In Scala 2, the syntax was a bit different. The condition was surrounded by parentheses, and
+there was no `do` keyword:
```

Comment:
I think it’s preferable to mention one reference syntax and to say that the alternative syntax is supported for compatibility reasons, primarily, otherwise I’m afraid readers will feel lost if we show them several ways of doing one thing.

Reply:
Sounds good. If we ever create the ability to show notes/tips/asides, I would put the Scala 2 text in one of those. (I’ve attached two images from the Cookbook to show what I mean by notes, tips, and asides.)

![tip](https://user-images.githubusercontent.com/309803/121818255-40fb0580-cc43-11eb-8d02-dd033fd33c0d.jpg)
![note](https://user-images.githubusercontent.com/309803/121818256-422c3280-cc43-11eb-9cd9-93f9dc043d06.jpg)